### ISSUE TITLE
Make it easy to search benchmarks

### ISSUE THREAD

**Issue by mrocklin** [coiled/coiled-runtime#399]
When I navigate to https://coiled.github.io/coiled-runtime/coiled-upstream-py3.9.html and look for a test like `test_basic_sum` I have to scroll through and look for `test_basic_sum` in a plot title.  Normal webpage search facilities don't track these though.  

Maybe we could add a subtle header before each test?

cc @ian-r-rose

**Comment by jrbourbeau** [comment]
+1

**Comment by gjoseph92** [comment]
Switching the vega-lite renderer to SVG would resolve this

**Comment by ian-r-rose** [comment]
> Switching the vega-lite renderer to SVG would resolve this

Right, that's what we did with the distributed test report. However, as far as I can see, Panel doesn't expose that option for its vega widget. So it's probably easier to just add an HTML tag with the title.
